### PR TITLE
Fix: illustration.js

### PR DESCRIPTION
**Pull request by timonsky**
- removed duplicate entry for 'Party' and 'Celebration'
- removed ambiguous terms 'Game' and 'Play' from matching with illustration of gaming controller [ game could also refer to any sports match, play could also be in an event title for a theater play ]
- added string 'Bike' to match with bicycle illustration

### CODE REVIEW

File: src/utils/illustration.js [hunk 1]
```diff
@@ -255,12 +237,15 @@ const data = [{
 		'Bicycle',
 		'Cycle',
 		'Biking',
+		'Bike'
```
Comment (ChristophWurst) [inline src/utils/illustration.js:240]: misses a `,` at the end

Comment (timonsky) [reply src/utils/illustration.js]: got it, thanks

**Review by tcitworld — APPROVED** [review]

**Comment by ChristophWurst** [comment]
@timonsky the DCO check fails. Could you please sign off your commits? Instructions can be found at https://github.com/nextcloud/calendar/pull/2962/checks?check_run_id=2168160848. Thanks.

**Comment by timonsky** [comment]
> @timonsky the DCO check fails. Could you please sign off your commits? Instructions can be found at https://github.com/nextcloud/calendar/pull/2962/checks?check_run_id=2168160848. Thanks.

Sorry, am totally new to github. I hope they are signed off. That first step seemed to work without an issue in the second step I got
```bash
$ git push --force-with-lease origin fix/illustration.js
remote: Permission to nextcloud/calendar.git denied to timonsky.
fatal: unable to access 'https://github.com/nextcloud/calendar.git/': The requested URL returned error: 403```

**Comment by timonsky** [comment]
Commits should now really be signed off

**Review by ChristophWurst — APPROVED** [review]

**Pull request merged**